mavros_msgs/msg/HilControls.msg

```diff
@@ -0,0 +1,16 @@
+# HilControls.msg
+#
+# ROS representation of MAVLink HIL_CONTROLS
+# See mavlink documentation
```

Comment:
Place link please. Looks like mavlink.org now down, but i hope it will be available soon.